src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -256,38 +259,37 @@ internal TabItem AddTabItem(IViewExtension viewExtension, ContentControl content
         private void CloseTab(object sender, RoutedEventArgs e)
         {
             string tabName = (sender as Button).CommandParameter.ToString();
-            CloseTab(tabName);
+
+            TabItem tabitem = TabItems.OfType<TabItem>().SingleOrDefault(n => n.Name == tabName);
+            CloseTab(tabitem);
```

Comment:
you should either check for null here or in `CloseTab` but it looks like you removed the null check?

Reply:
Added it.